app/nginx/scripts/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+/scripts/get-certs.sh
+
+# start cron to automatically renew certs every 5 days
```

Comment:
letsencrypt issues 90-day certs I believe.

Reply:
Yes, it's just good to check them frequently. This just checks the expiry time and exits if it's not due for renewal (~30 days before expiry, I think)

Reply:
You are right, then every 5 days are perfect. Resolving